shared/src/assets/icons/defiApp.tsx

```diff
@@ -40,12 +40,42 @@ export const YearnLogo: React.FC<SVGProps> = (props) => (
   </svg>
 );
 
-export const GnosisLogo: React.FC<SVGProps> = (props) => (
-  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2000 2000" {...props}>
-    <g fill="#00a6c4">
-      <path d="M1809.4 500l-73.6 73.6c58.5 92 76.9 207.4 36.8 317.7-66.9 187.3-274.2 286-463.2 219.1-26.8-10-51.8-21.7-73.6-36.8L1000 1309.4 794.3 1102c-95.3 63.5-217.4 81.9-332.8 41.8-194-70.2-294.3-282.6-224.1-474.9 10-30.1 25.1-56.9 41.8-81.9l-88.6-88.6-16.7 28.4c-92 150.5-142.1 324.4-142.1 503.3-1.7 533.4 433.1 969.9 966.5 969.9h1.7c533.4 0 966.6-433.1 968.2-966.6 0-177.3-48.5-351.2-140.5-503.3l-18.3-30.1" />
-      <path d="M388 695.7c-25.1 33.4-40.1 75.3-40.1 120.4 0 108.7 88.6 197.3 197.3 197.3 45.2 0 87-15.1 120.4-41.8L388 695.7M1356.2 954.8c31.8 21.7 68.6 33.4 110.4 33.4 108.7 0 197.3-88.6 197.3-197.3 0-40.1-11.7-78.6-33.4-110.4l-274.3 274.3M1003.3 1162.2L204 359.5l31.8-33.4C433.1 117.1 702.3 0 991.6 0h1.7c292.6 0 573.6 125.4 769.2 342.8l30.1 33.4-789.3 786M334.5 359.5l668.9 670.6 660.5-657.2C1486.6 195.7 1245.8 93.6 995 93.6h-1.7c-249.2 0-481.6 93.7-658.8 265.9" />
-    </g>
+export const ParadigmLogo: React.FC<SVGProps> = (props) => (
+  <svg
+    width="56"
+    height="56"
+    viewBox="0 0 56 56"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M21.8875 52C31.7666 52 39.7751 44.0582 39.7751 34.2614C39.7751 24.4647 31.7666 16.5229 21.8875 16.5229C12.0085 16.5229 4 24.4647 4 34.2614C4 44.0582 12.0085 52 21.8875 52Z"
+      fill={props.color}
+    />
+    <path
+      d="M52 27.5754C52 40.2015 41.7332 50.4264 29.0584 50.4264C16.3837 50.4264 6.11687 40.2015 6.11687 27.5754C6.11687 14.9492 16.3837 4.72437 29.0584 4.72437C41.7332 4.72437 52 14.9492 52 27.5754Z"
+      fill="url(#paint0_radial_801_59)"
+      fill-opacity="0.5"
```

Comment:
`fill-opacity`, `stop-color`, `stop-opacity` needs to be updated to camel case to prevent console errors